lib/v2/domain-shared/bloc_observer.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:bloc/bloc.dart';
+
+class SimpleBlocObserver extends BlocObserver {
```

Comment:
can you leave a comment on this class explaining what is this used for.

Reply:
Of curse 👍